Makefile

```diff
@@ -15,6 +14,7 @@ endif
 all: fast-docker-build
 
 docker-build:
+	git show -s --format="COMMITTED_AT=%ai%nCOMMIT_HASH=%h%n" HEAD > hack/coreapi-release
```

Comment:
It is not clear what you are doing with this file.

Reply:
This is used by one of the api(v1/version) endpoint to return the image created date and commit hash. Earlier this was run inside Dockerfile after copying .git files into image, which is an expensive affair.

Reply:
Why do we need those details exposed by an API?  Is there a consumer for it?

Reply:
I'm not sure why it exists. Just don't want to break that api as part of this PR. We can remove it latter if we want to.

https://recommender.api.prod-preview.openshift.io/api/v1/system/version